BingMaps/coverage/transit-coverage/south-america.md

```diff
@@ -63,7 +63,7 @@ Bing Maps APIs provide transit information for the following transit agencies in
 |Florianopolis|[Consórcio Fênix](https://www.consorciofenix.com.br) 
 |Fortaleza|[COMPANHIA CEARENSE DE TRANSPORTES METROPOLITANOS](http://www.metrofor.ce.gov.br) 

```

Comment:
I got 

Hmmm… can't reach this page
www.metrofor.ce.gov.br refused to connect.